app/models/bus_stops_route.rb

```diff
@@ -0,0 +1,7 @@
+class BusStopsRoute < ApplicationRecord
+  belongs_to :bus_stop
+  belongs_to :route
+
+  validates :sequence, presence: true, numericality: { greater_than_or_equal_to: 1 }
```

Comment:
Sequence should also be unique relative to route and direction, correct?

Also bus stop ID, should be unique relative to route and direction.

Reply:
Like I did below?

Reply:
I hope I did it right

Reply:
Did it right. Tested.